modules/theme/src/themes/default/assets/images/illustrations/federated-associations.svg

```diff
@@ -0,0 +1,52 @@
+        <path class="path stroke white" stroke-width="1" d="M147.468+55.8023C143.038+55.8023+139.345+58.7341+138.062+62.7383C136.874+61.0859+135.036+59.9323+132.845+59.9323C129.23+59.9323+126.304+62.8582+126.304+66.473C126.304+67.2232+126.507+67.9163+126.739+68.5874C124.326+70.1659+122.728+72.8788+122.728+75.9779C122.728+80.8606+126.678+84.8109+131.561+84.8109L155.57+84.8109C160.452+84.8109+164.423+80.8606+164.423+75.9779C164.423+71.672+161.325+68.1101+157.249+67.3227C157.332+66.8077+157.407+66.2802+157.407+65.7419C157.407+60.2526+152.957+55.8023+147.468+55.8023Z" fill="#34a850" stroke-linecap="round" opacity="1" stroke-linejoin="round"/>
+    </g>
+    <g id="connection">
+        <path stroke="#d6d6d6" stroke-width="1.92" d="M198.615+24.5138L198.615+24.5138C202.017+24.5138+204.775+27.272+204.775+30.6745L204.775+30.6745C204.775+34.0769+202.017+36.8351+198.615+36.8351L198.615+36.8351C195.212+36.8351+192.454+34.0769+192.454+30.6745L192.454+30.6745C192.454+27.272+195.212+24.5138+198.615+24.5138ZM175.068+14.7363L175.068+14.7363C176.741+14.7363+178.097+16.0927+178.097+17.7658L178.097+17.7658C178.097+19.439+176.741+20.7954+175.068+20.7954L175.068+20.7954C173.395+20.7954+172.038+19.439+172.038+17.7658L172.038+17.7658C172.038+16.0927+173.395+14.7363+175.068+14.7363ZM180.631+41.6797L180.631+41.6797C183.623+41.6797+186.049+44.1054+186.049+47.0977L186.049+47.0977C186.049+50.09+183.623+52.5157+180.631+52.5157L180.631+52.5157C177.639+52.5157+175.213+50.09+175.213+47.0977L175.213+47.0977C175.213+44.1054+177.639+41.6797+180.631+41.6797ZM213.958+13.3764L213.958+13.3764C215.864+13.3764+217.409+14.9213+217.409+16.8269L217.409+16.8269C217.409+18.7326+215.864+20.2775+213.958+20.2775L213.958+20.2775C212.053+20.2775+210.508+18.7326+210.508+16.8269L210.508+16.8269C210.508+14.9213+212.053+13.3764+213.958+13.3764ZM216.341+38.1251L216.341+38.1251C218.471+38.1251+220.198+39.8522+220.198+41.9828L220.198+41.9828C220.198+44.1133+218.471+45.8404+216.341+45.8404L216.341+45.8404C214.21+45.8404+212.483+44.1133+212.483+41.9828L212.483+41.9828C212.483+39.8522+214.21+38.1251+216.341+38.1251ZM184.183+42.7801L193.365+33.5213M176.961+20.2964L192.647+29.67M203.424+26.6416L211.892+19.373M203.139+35.0612L213.205+40.529M198.146+36.9307M198.427+36.9187L202.824+56.5205M203.959+56.2944L203.959+56.2944C207.431+56.2944+210.245+59.1088+210.245+62.5805L210.245+62.5805C210.245+66.0523+207.431+68.8666+203.959+68.8666L203.959+68.8666C200.487+68.8666+197.673+66.0523+197.673+62.5805L197.673+62.5805C197.673+59.1088+200.487+56.2944+203.959+56.2944ZM197.941+24.3772L197.4+16.1105M197.334+7.30308L197.334+7.30308C199.783+7.30308+201.769+9.2999+201.769+11.7631L201.769+11.7631C201.769+14.2263+199.783+16.2231+197.334+16.2231L197.334+16.2231C194.886+16.2231+192.9+14.2263+192.9+11.7631L192.9+11.7631C192.9+9.2999+194.886+7.30308+197.334+7.30308Z" fill="#3f6b92" stroke-linecap="round" fill-opacity="0" opacity="1" stroke-linejoin="round"/>
```

Comment:
Can we take the hard code colors out? Else it will not adopt to central theming.